src/types/Status.ts

```diff
@@ -1 +1,5 @@
-export type Status = 'all' | 'active' | 'completed';
+export enum Status {
+  all = 'all',
+  active = 'active',
+  completed = 'completed',
```

Comment:
```suggestion
export enum Status {
  All = 'all',
  Active = 'active',
  Completed = 'completed',
```
Enum properties should be in PascalCase